docker/entrypoint-unit-tests.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 # Run available unittests with a simple setup
+umask 0002
```

Comment:
I don't think you need this as you use the release mode, so it's using entrypoint-uwsgi.sh

Reply:
yea...i can remove that...was added when i was really trying to find out how to fix the issue i had.

Reply:
removed..